code/modules/vehicles/wheelchair.dm

```diff
@@ -14,6 +14,25 @@
 	var/image/wheels_overlay
 	///Determines the typepath of what the object folds into
 	var/foldabletype = /obj/item/wheelchair
+	///Determines wether the wheelchair has a bell on it or not
+	var/bell_attached
+
+/obj/vehicle/ridden/wheelchair/generate_actions()
+	if(!bell_attached)
+		return
+	. = ..()
+	initialize_controller_action_type(/datum/action/vehicle/ridden/wheelchair/bell, VEHICLE_CONTROL_DRIVE)
+
+/obj/vehicle/ridden/wheelchair/proc/attach_bell(obj/structure/desk_bell/bell)
+	bell_attached = bell
+	src.generate_actions()
+	icon_state = "wheelchair_bell"
+	desc += "There is a small bell attached to the handle."
+
+/obj/vehicle/ridden/wheelchair/gold/attach_bell()
+	..()
+	icon_state = "gold_wheelchair_bell"
```

Comment:
I'd recommend using an overlay rather than making separate icon_states for gold and normal wheelchairs.

Reply:
> I'd recommend using an overlay rather than making separate icon_states for gold and normal wheelchairs.

thx i dont know shit about dick in terms of spriting